src/txmempool.h

```diff
@@ -78,15 +79,20 @@ class CTxMemPoolEntry
 
 public:
     CTxMemPoolEntry(const CTransaction& _tx, const CAmount& _nFee,
-                    int64_t _nTime, double _dPriority, unsigned int _nHeight, bool poolHasNoInputsOf = false);
+                    int64_t _nTime, double _entryPriority, unsigned int _entryHeight,
+                    bool poolHasNoInputsOf, CAmount _inChainInputValue);
     CTxMemPoolEntry(const CTxMemPoolEntry& other);
 
     const CTransaction& GetTx() const { return this->tx; }
+    /**
+     * Fast calculation of lower bound of current priority as update
+     * from entry priority.  Only originally in-chain inputs will age
+     */
```

Comment:
How about you change it to "Only inputs that were originally in-chain will age."

On 20 November 2015 16:34:30 GMT-05:00, Alex Morcos <notifications@github.com> wrote:
>>      CTxMemPoolEntry(const CTxMemPoolEntry& other);
>>  
>>      const CTransaction& GetTx() const { return this->tx; }
>> +    /**
>> +     * Fast calculation of lower bound of current priority as update
>> +     * from entry priority.  Only originally in-chain inputs will
>age
>> +     */
>
>I'm happy to make it clearer, but that was the intended wording except
>for perhaps a missing ".".
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/bitcoin/bitcoin/pull/7008/files#r45521266